pkg/detectors/flightlabs/flightlabs.go

```diff
@@ -0,0 +1,84 @@
+package flightlabs
+
+import (
+	"context"
+	"io/ioutil"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct{}
+
+// Ensure the Scanner satisfies the interface at compile time
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	//Make sure that your group is surrounded in boundry characters such as below to reduce false positives
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"flightlabs"}) + `\b([a-zA-Z-0-9.-_]{359})\b`)
```

Comment:
suggestion: this regex can be made a bit stricter by adding `ey` as it always starts with it `\b(ey[a-zA-Z0-9]{34}.ey[a-zA-Z0-9._-]{300,350})` and they are all split into two section by `.ey` separator so we can use that to our advantage. Also added a range to it since it is in jwt encoding.

Reply:
Hi Roxanne, fixed the regex. Thank you!